{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"struct"},{"kind":"text","text":" "},{"kind":"identifier","text":"InternalServiceError"}],"languages":["swift"],"platforms":["macOS"]}]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/awsappintegrations\/internalserviceerror"],"traits":[{"interfaceLanguage":"swift"}]}],"relationshipsSections":[{"identifiers":["doc:\/\/AWSAppIntegrations\/16AWSClientRuntime19AWSHttpServiceErrorP","doc:\/\/AWSAppIntegrations\/13ClientRuntime16HttpServiceErrorP","doc:\/\/AWSAppIntegrations\/13ClientRuntime12ServiceErrorP","doc:\/\/AWSAppIntegrations\/SQ"],"kind":"relationships","title":"Conforms To","type":"conformsTo"}],"identifier":{"url":"doc:\/\/AWSAppIntegrations\/documentation\/AWSAppIntegrations\/InternalServiceError","interfaceLanguage":"swift"},"abstract":[{"type":"text","text":"Request processing failed due to an error or failure with the service."}],"kind":"symbol","metadata":{"fragments":[{"kind":"keyword","text":"struct"},{"kind":"text","text":" "},{"kind":"identifier","text":"InternalServiceError"}],"title":"InternalServiceError","roleHeading":"Structure","role":"symbol","symbolKind":"struct","externalID":"s:18AWSAppIntegrations20InternalServiceErrorV","modules":[{"name":"AWSAppIntegrations"}],"navigatorTitle":[{"kind":"identifier","text":"InternalServiceError"}]},"hierarchy":{"paths":[["doc:\/\/AWSAppIntegrations\/documentation\/AWSAppIntegrations"]]},"topicSections":[{"title":"Initializers","identifiers":["doc:\/\/AWSAppIntegrations\/documentation\/AWSAppIntegrations\/InternalServiceError\/init(httpResponse:decoder:message:requestID:)","doc:\/\/AWSAppIntegrations\/documentation\/AWSAppIntegrations\/InternalServiceError\/init(message:)"]},{"title":"Instance Properties","identifiers":["doc:\/\/AWSAppIntegrations\/documentation\/AWSAppIntegrations\/InternalServiceError\/message"]},{"title":"Default Implementations","identifiers":["doc:\/\/AWSAppIntegrations\/documentation\/AWSAppIntegrations\/InternalServiceError\/Equatable-Implementations"],"generated":true}],"references":{"doc://AWSAppIntegrations/documentation/AWSAppIntegrations/InternalServiceError/Equatable-Implementations":{"role":"collectionGroup","title":"Equatable Implementations","abstract":[],"identifier":"doc:\/\/AWSAppIntegrations\/documentation\/AWSAppIntegrations\/InternalServiceError\/Equatable-Implementations","kind":"article","type":"topic","url":"\/documentation\/awsappintegrations\/internalserviceerror\/equatable-implementations"},"doc://AWSAppIntegrations/documentation/AWSAppIntegrations/InternalServiceError/message":{"role":"symbol","title":"message","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"message"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":"?"}],"abstract":[],"identifier":"doc:\/\/AWSAppIntegrations\/documentation\/AWSAppIntegrations\/InternalServiceError\/message","kind":"symbol","type":"topic","url":"\/documentation\/awsappintegrations\/internalserviceerror\/message"},"doc://AWSAppIntegrations/documentation/AWSAppIntegrations/InternalServiceError":{"role":"symbol","title":"InternalServiceError","fragments":[{"kind":"keyword","text":"struct"},{"kind":"text","text":" "},{"kind":"identifier","text":"InternalServiceError"}],"abstract":[{"type":"text","text":"Request processing failed due to an error or failure with the service."}],"identifier":"doc:\/\/AWSAppIntegrations\/documentation\/AWSAppIntegrations\/InternalServiceError","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"InternalServiceError"}],"url":"\/documentation\/awsappintegrations\/internalserviceerror"},"doc://AWSAppIntegrations/16AWSClientRuntime19AWSHttpServiceErrorP":{"type":"unresolvable","title":"AWSClientRuntime.AWSHttpServiceError","identifier":"doc:\/\/AWSAppIntegrations\/16AWSClientRuntime19AWSHttpServiceErrorP"},"doc://AWSAppIntegrations/documentation/AWSAppIntegrations/InternalServiceError/init(httpResponse:decoder:message:requestID:)":{"role":"symbol","title":"init(httpResponse:decoder:message:requestID:)","fragments":[{"kind":"identifier","text":"init"},{"kind":"text","text":"("},{"kind":"externalParam","text":"httpResponse"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"HttpResponse","preciseIdentifier":"s:13ClientRuntime12HttpResponseC"},{"kind":"text","text":", "},{"kind":"externalParam","text":"decoder"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"ResponseDecoder","preciseIdentifier":"s:13ClientRuntime15ResponseDecoderP"},{"kind":"text","text":"?, "},{"kind":"externalParam","text":"message"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":"?, "},{"kind":"externalParam","text":"requestID"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":"?) "},{"kind":"keyword","text":"throws"}],"abstract":[],"identifier":"doc:\/\/AWSAppIntegrations\/documentation\/AWSAppIntegrations\/InternalServiceError\/init(httpResponse:decoder:message:requestID:)","kind":"symbol","type":"topic","url":"\/documentation\/awsappintegrations\/internalserviceerror\/init(httpresponse:decoder:message:requestid:)"},"doc://AWSAppIntegrations/documentation/AWSAppIntegrations":{"role":"collection","title":"AWSAppIntegrations","abstract":[],"identifier":"doc:\/\/AWSAppIntegrations\/documentation\/AWSAppIntegrations","kind":"symbol","type":"topic","url":"\/documentation\/awsappintegrations"},"doc://AWSAppIntegrations/13ClientRuntime16HttpServiceErrorP":{"type":"unresolvable","title":"ClientRuntime.HttpServiceError","identifier":"doc:\/\/AWSAppIntegrations\/13ClientRuntime16HttpServiceErrorP"},"doc://AWSAppIntegrations/documentation/AWSAppIntegrations/InternalServiceError/init(message:)":{"role":"symbol","title":"init(message:)","fragments":[{"kind":"identifier","text":"init"},{"kind":"text","text":"("},{"kind":"externalParam","text":"message"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":"?)"}],"abstract":[],"identifier":"doc:\/\/AWSAppIntegrations\/documentation\/AWSAppIntegrations\/InternalServiceError\/init(message:)","kind":"symbol","type":"topic","url":"\/documentation\/awsappintegrations\/internalserviceerror\/init(message:)"},"doc://AWSAppIntegrations/SQ":{"type":"unresolvable","title":"Swift.Equatable","identifier":"doc:\/\/AWSAppIntegrations\/SQ"},"doc://AWSAppIntegrations/13ClientRuntime12ServiceErrorP":{"type":"unresolvable","title":"ClientRuntime.ServiceError","identifier":"doc:\/\/AWSAppIntegrations\/13ClientRuntime12ServiceErrorP"}}}